utils.css

```diff
@@ -34,18 +54,50 @@
     position: absolute;
 }
 
+.mb-0 {
+    margin-bottom: 0;
+}
+
+.padding-02 {
+    padding: 0 2em;
+}
+
+.padding-left3 {
+    padding-left: 3rem;
+}
+
+.font-size18px {
+    font-size: 18px;
+}
+
 .font-12 {
     font-size: 12px;
 }
 
+.font-14 {
+    font-size: 14px;
+}
+
+.font-15 {
+    font-size: 15px;
+}
+
+.font-weight-light {
+    font-weight: light;
+}
+
 .mr-03 {
     margin-right: 0.3em;
 }
 
-.mb-0 {
-    margin-bottom: 0;
+.ml1 {
+    margin-left: 1em;
+}
+
+.pad-0508 {
```

Comment:
We should split into X and Y axis. Have one rule

`px-2` = 0.5
`py-3` = 0.75

Then when you want to add both, you simply add both classes. `<a class="px-2 py-3"`.